core/services/relay/evm/mercury/transmitter.go

```diff
@@ -388,15 +388,22 @@ func (mt *mercuryTransmitter) HealthReport() map[string]error {
 	return report
 }
 
-func (mt *mercuryTransmitter) sendToTrigger(report ocrtypes.Report, signatures []ocrtypes.AttributedOnchainSignature) error {
+func (mt *mercuryTransmitter) sendToTrigger(report ocrtypes.Report, rawReportCtx [3][32]byte, signatures []ocrtypes.AttributedOnchainSignature) error {
```

Comment:
Shouldn't this `rawReportCtx [3][32]byte,` be based on a workflow spec config?

Reply:
How so? The context encodes OCR round and epoch in which the report was produced.

Reply:
I guess it's not clear what this context is.

Reply:
It should be pretty clear in the context of the Transmitter because it already uses it elsewhere.